docs/source/model/dependencies.rst

```diff
@@ -438,15 +445,15 @@ To do so, use the **pip-requirements-override** option to specify pip dependenci
         import mlflow
 
         mlflow.models.predict(
-            model_uri="runs:/<run_id>/model",
+            model_uri="runs:/<run_id>/<model_path>",
             input_data=<input_data>,
```

Comment:
Could you add similar 1~3 sentences description about the input data in this tab for Python?